src/pages/TabDetails.tsx

```diff
@@ -86,89 +101,110 @@ export function useInfoForOneCycle(idx: number): InfoOneCycle {
   };
 }
 
+const lenCycleStyle = {
+  fontSize: "13px" as const,
+  color: "var(--ion-color-black)" as const,
+  textAlign: "left" as const,
+};
+
+const datesStyle = {
+  fontSize: "11px" as const,
+  color: "var(--ion-color-medium)" as const,
+  textAlign: "left" as const,
+};
+
 const CurrentCycle = () => {
+  const { t } = useTranslation();
   const title = useTitleLastCycle();
   const progressBarBuffer = useProgressBarBuffer();
 
   const cycles = useContext(CyclesContext).cycles;
   const startDate = getLastStartDate(cycles);
   const lengthOfPeriod = getAverageLengthOfPeriod(cycles);
 
-  const { t } = useTranslation();
-
   return (
-    <IonItem
-      class="transparent-center"
-      lines="none"
-    >
-      <IonLabel position="stacked">
-        {title ? (
-          <h2>{`${t("Current cycle")}: ${title}`}</h2>
-        ) : (
-          <h2>{t("Current cycle")}</h2>
-        )}
-      </IonLabel>
-      <IonLabel position="stacked">
-        {startDate ? (
-          <p>{`${t("Started date")} ${new Date(
-            startDate,
-          ).toLocaleDateString()}`}</p>
-        ) : (
-          <p>{t("Started date")}</p>
-        )}
-      </IonLabel>
-      <IonLabel position="stacked">
+    <div id="progress-block">
+      <div style={{ marginLeft: "15px" }}>
+        <IonLabel>
+          <p style={lenCycleStyle}>{title}</p>
+        </IonLabel>
         <IonProgressBar
           class="current-progress"
+          style={{ marginTop: "5px", marginBottom: "5px" }}
           value={(lengthOfPeriod / 100) * 3}
           buffer={(progressBarBuffer / 100) * 3}
-        ></IonProgressBar>
-      </IonLabel>
-    </IonItem>
+        />
+        <IonLabel>
+          {t("locale") === "ru" ? (
+            <p style={datesStyle}>{`${format(new Date(startDate), "dd MMMM", {
```

Comment:
And same here. You placed locale parsing inside this component and if we'll introduce a new language this block will be too compilcated

Reply:
Did it.